lib/css/exports/constants-colors.less

```diff
@@ -197,6 +197,10 @@
     --theme-link-color: var(--theme-secondary-400);
     --theme-link-color-hover: var(--theme-secondary-350);
     --theme-link-color-visited: var(--theme-secondary-500);
+    // Below variables set as theme override in consumers
+    // --theme-link-color-dark: undefined
+    // --theme-link-color-dark-hover: undefined
+    // --theme-link-color-dark-visited: undefined
```

Comment:
I'd rather not set a precedence for this. If consumers want to override specific variables for specific modes, then they can do so with our mode mixins.

For example:

```less
.dark-mode({
--theme-link-color: red;
})
```

Reply:
Yeah, that sounds reasonable. I'm trying (and failing) to recall why I went about it how I did. If I can remember and it's worth considering, I'll mention it here. Otherwise, I'm good to remove these dark-specific custom properties.

Reply:
Ah, my memory came through!

 I think we have a naming issue here. These variables have nothing to do with the _dark_ theme specifically. They're intended to define a color to use when links are rendered on a non-standard background (think watched or deleted post summaries).

I'd still be open to reconsidering this approach altogether, but the surface issue here is my use of the term `dark`. Maybe a better alternative would be:

```
    // Below variables set as theme override in consumers
    // --theme-link-color-emphasised: undefined
    // --theme-link-color-emphasised-hover: undefined
    // --theme-link-color-emphasised-visited: undefined
```

Reply:
I've removed those variables for now. We can punt and try to get this one across the finish line.